### DISCUSSION TITLE
JS Promises 🤞

### DISCUSSION THREAD

**Discussion by TrinityChristiana** [nss-evening-cohort-14/evening-client-side#137] [Class Notes and Discussion]
### JavaScript Promise Object definition with an axios call
<img width="800" alt="JavaScript Promise Object definition with an axios call" src="https://user-images.githubusercontent.com/31781724/108935702-cb2b4980-7613-11eb-9af2-10a6991ddb2e.png">

### New JavaScript Promise with an Axios call and rejecting error in a catch
<img width="800" alt="New Promise with Axios call, and rejecting in a catch" src="https://user-images.githubusercontent.com/31781724/108935701-ca92b300-7613-11eb-8144-8505a96fd305.png">

### New JavaScript Promise with an Axios call, rejecting error in a catch, and resolving an error in a then method
<img width="800" alt="New JavaScript Promise with an Axios call, rejecting error in a catch, and resolving an error in a then method" src="https://user-images.githubusercontent.com/31781724/108935698-ca92b300-7613-11eb-9e2f-917e106c05b5.png">

**Comment by drteresavasquez** [comment]
Keep this in mind for when we move forward tomorrow.

**Comment by drteresavasquez** [comment]
# Promise.all() Example
<img width="1151" alt="Screen Shot 2021-03-06 at 1 05 49 PM" src="https://user-images.githubusercontent.com/29741570/110217973-bb65fd80-7e7c-11eb-980b-b1db74991dad.png">

**Comment by drteresavasquez** [comment]
@nss-evening-cohort-14/students remember to keep your promises!!!! 💯
